src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    const [property, ...valueParts] = declaration.split(':');
+
+    if (!property || valueParts.length === 0) {
+      return;
+    }
+
+    const value = valueParts.join(':');
+
+    const trimmedProperty = property.trim();
+    const trimmedValue = value.trim();
+
+    if (trimmedProperty === '' || trimmedValue === '') {
+      return;
+    }
+    stylesObject[trimmedProperty] = trimmedValue;
```

Comment:
[CODE STYLE] - The chained method calls should have the same indentation to improve readability. Consider formatting the chain of operations on the 'declarations' array with proper indentation.